src/main.spec.ts

```
@@ -0,0 +1,54 @@
+    expect(defaultUrl).toBe('http://localhost:3000');
+
+    process.env.BACKEND_URL = '';
+    const emptyUrl = process.env.BACKEND_URL ?? 'http://localhost:3000';
+    expect(emptyUrl).toBe('');
+
+    process.env.BACKEND_URL = originalEnv;
+  });
+
+  it('should validate bootstrap execution logic', () => {
+    const originalEnv = process.env.BACKEND_URL;
+    delete process.env.BACKEND_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)